packages/common/react-components/src/components/Input/Input.tsx

```diff
@@ -56,7 +57,7 @@ export const Input = ({
     ...(autoFocus && !hasIosKeyboard && { autoFocus: true }),
     disabled,
     placeholder,
-    value,
+    value: value ?? '',
```

Comment:
In some situations this has locked the input’s value at empty for me, I'd just make sure that isn’t happening here.

Reply:
I've used this pattern for a long time and not seen that but i will watch out for that.